NuguAgents/Sources/CapabilityAgents/AutomaticSpeechRecognition/ASRAgent.swift

```diff
@@ -268,7 +269,12 @@ public extension ASRAgent {
                 return
             }
          
-            self.startRecognition(initiator: initiator, eventIdentifier: eventIdentifier, completion: completion)
+            self.startRecognition(
```

Comment:
self 는 사용하지 않는 것이 좋겠습니다
수정하는 코드에는 적용 부탁드려요

```suggestion
            startRecognition(
```